### ISSUE TITLE
@ngrx/data - master branch - ERROR in node_modules/@ngrx/data/src/entity-data.module.d.ts:12:22 - error NG6002: Appears in the NgModule.imports of AppStoreModule, but could not be resolved to an NgModule class

### ISSUE THREAD

**Issue by jonstelly** [ngrx/platform#2384]
## Minimal reproduction of the bug/regression with instructions:
I'm testing out moving my application to angular 9 after I saw the @ngrx 9.0 beta.  I couldn't get it running with those packages, tried building directly against the ngrx/data-builds, store-builds, repositories, and that's not working either.

1. clone: https://github.com/jonstelly/ng-resolve-module-class
2. `npm i`
3. `ng serve`

```
ERROR in node_modules/@ngrx/data/src/entity-data.module.d.ts:12:22 - error NG6002: Appears in the NgModule.imports of AppStoreModule, but could not be resolved to an NgModule class

12 export declare class EntityDataModule {
                        ~~~~~~~~~~~~~~~~
src/app/app-store.module.ts:22:16 - error NG6002: Appears in the NgModule.imports of AppModule, but itself has errors

22   export class AppStoreModule {}
                  ~~~~~~~~~~~~~~
```
## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):
```
Angular CLI: 9.0.2
Node: 12.11.1
OS: win32 x64
```

## I would be willing to submit a PR to fix this issue

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by jonstelly** [comment]
I had searched before creating this issue but didn't see anything... But I just saw this other issue: https://github.com/ngrx/platform/issues/2367

It's technically a different angular cli error (NG6002 vs TS-996002) so I'm not sure if it's the exact same issue but we can close it and track there if you'd prefer.

**Comment by jonstelly** [comment]
If it helps at all, I took all of the @ngrx code from master and threw it into my example's repository and got things working.

I created libraries using the angular cli, then replaced source code for each of them with the code from ngrx/platform.  It's in that same repository under the [local-ngrx-build branch](https://github.com/jonstelly/ng-resolve-module-class/tree/local-ngrx-build).

You can build and run the sample app with the following powershell.  I realize this may not be helpful in itself, but it seems to suggest the issues are packaging issues, not necessarily anything wrong with the ngrx code.

```
npm i
@('store', 'effects', 'entity', 'data', 'router-store', 'store-devtools') | % { ng build "@ngrx/$_" }
ng serve
```

**Comment by timdeschryver** [comment]
This seems like a duplicate of #2367 ?

**Comment by timdeschryver** [comment]
Closed by #2387
